src/repositories/organisations/inmemory.test.js

```diff
@@ -15,6 +15,25 @@ describe('In-memory organisations repository', () => {
     expect(result).toEqual(initial)
   })
 
+  it('findByOrgId returns matching organisation', async () => {
+    const initial = [
+      { _id: 'mongo-1', orgId: 500123, name: 'Acme' },
```

Comment:
Seems weird exposing mongodb-specific stuff in the domain model (if the domain model is what it's returning here?).